config/environments/default.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+# Default environment file to configure the base project where everything
+# is relative to the project folder.  This configuration should always
+# work for everyone attempting to reproduce this research without having
+# to make any changes.
```

Comment:
Would it be useful to add a note not to edit this file?

Reply:
I am not sure if my instructions were clear so I expanded on them.  I added a warning that reviewers should not have to change the file for the benefit of the author and reviewers along with a more detailed explanation.

The author should change this file in ways that reviewers do not have to change it.